src/Http/Http.Features/src/HttpsCompressionMode.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Http.Features
+{
+    /// <summary>
+    /// Use to dynamically control response compression for HTTPS requests.
+    /// </summary>
+    public enum HttpsCompressionMode
+    {
+        /// <summary>
+        /// No value has been specified, use the configured defaults.
```

Comment:
Specify what the default is.

Reply:
The default is specified on the properties where this is used.